simple_scheduler/jobs/sample_job.py

```diff
@@ -22,6 +22,7 @@ def meta_info(cls):
 
     def run(self, argument1, argument2, *args, **kwargs):
         print('Hello from AwesomeJob! Argument1: %s, Argument2: %s' % (argument1, argument2))
+        return [argument1, argument2]
```

Comment:
Could you also change the block comment here https://github.com/Nextdoor/ndscheduler/blob/master/ndscheduler/job.py#L116

Make it clear that the return value of this function will be stored in database and be shown as result in UI.